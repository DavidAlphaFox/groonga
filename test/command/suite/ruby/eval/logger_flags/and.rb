#@on-error omit
plugin_register ruby/eval
#@on-error default

ruby_eval "(Groonga::Logger::Flags.new([:time, :thread_id]) & [:location, :time]).to_s"
