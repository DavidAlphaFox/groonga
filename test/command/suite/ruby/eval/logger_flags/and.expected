plugin_register ruby/eval
[[0,0.0,0.0],true]
ruby_eval "(Groonga::Logger::Flags.new([:time, :thread_id]) & [:location, :time]).to_s"
[[0,0.0,0.0],{"value":"time"}]
